src/master/develop/sql/transactions.md

```diff
@@ -1,26 +1,34 @@
 # Transactions
 
+::: tip
+Examples in multiple languages can be found at following links: [immudb SDKs examples](https://github.com/codenotary/immudb-client-examples)
+:::
+
+<WrappedSection>
+
 immudb supports transactions both on key-value and SQL level, but interactive transactions are supported only on SQL with the exception of `execAll` method, that provides some additional properties.
 
 Interactive transactions are a way to execute multiple SQL statements in a single transaction. This makes possible to delegate application logic to SQL statements - a very common use case is for example checking if the balance > 0 before making a purchase.
 In order to create a transaction, you must call the `NewTx()` method on the client instance. The resulting object is a transaction object that can be used to execute multiple SQL statements, queries, commit or rollback.
 Following there are methods exposed by the transaction object:
 
-<WrappedSection>
+</WrappedSection>
 
-```
+```go
```

Comment:
The go example should be inside wrappedsection, currently it looks like that:

![image](https://user-images.githubusercontent.com/1365626/176174428-41248f38-c8e9-480f-95db-91ea7bd47647.png)